docs/src/modules/components/MaterialYouUsageDemo.tsx

```diff
@@ -93,11 +93,11 @@ export const prependLinesSpace = (code: string, size: number = 2) => {
   return newCode.join('\n');
 };
 
-function ModeSwitcher({ md2Mode }: { md2Mode: Mode | undefined }) {
-  const { mode, setMode } = useColorScheme();
-  if (md2Mode && mode !== md2Mode) {
-    setMode(md2Mode ?? mode);
```

Comment:
I found that this line `setState` at the render phase which is not allowed in React.